server/internal/storage/gorm/client.go

```diff
@@ -163,36 +137,36 @@ func (c *Client) IsNotFound(err error) bool {
 }
 
 // Get gets an entity using the storage client.
-func (c *Client) Get(ctx context.Context, k storage.Key, v interface{}) error {
-	mylock()
-	defer myunlock()
-	return c.db.Where("key = ?", k.(*Key).Name).First(v).Error
+func (c *Client) Get(ctx context.Context, k *Key, v interface{}) error {
+	lock()
+	defer unlock()
+	return c.db.Where("key = ?", k.Name).First(v).Error
 }
 
 // Put puts an entity using the storage client.
-func (c *Client) Put(ctx context.Context, k storage.Key, v interface{}) (storage.Key, error) {
-	mylock()
-	defer myunlock()
+func (c *Client) Put(ctx context.Context, k *Key, v interface{}) (*Key, error) {
```

Comment:
Not for this PR: I expect that we'll eventually flatten out this artifact of using Cloud Datastore